simpleclient/src/main/java/io/prometheus/client/Gauge.java

```diff
@@ -181,6 +181,21 @@ public void setToCurrentTime() {
     public Timer startTimer() {
       return new Timer(this);
     }
+
+    /**
+     * Executes runnable code (ie Java 8 Lambda) and observes a duration of how long it took to run.
+     *
+     * @param timeable Code that is being timed
+     */
+    public void time(Runnable timeable){
```

Comment:
These need to be added to the convenience functions too, so they'll work as expected for metrics without labels.